base/multidimensional.jl

```diff
@@ -377,7 +377,7 @@ module IteratorsMD
     next(iter::Reverse{<:CartesianIndices{0}}, state) = CartesianIndex(), true
     done(iter::Reverse{<:CartesianIndices{0}}, state) = state
 
-    LinearIndices(inds::CartesianIndices{N,R}) where {N,R} = LinearIndices{N,R}(inds.indices)
+    Base.LinearIndices(inds::CartesianIndices{N,R}) where {N,R} = LinearIndices{N,R}(inds.indices)
```

Comment:
This was a module local function, which was undocumented. Documenter then complained that there were no docs, since we reference this function in the same module (L214).